Source/WebKitLegacy/mac/DOM/DOMElement.h

```diff
@@ -89,8 +89,6 @@ WEBKIT_CLASS_DEPRECATED_MAC(10_4, 10_14)
 - (void)blur WEBKIT_AVAILABLE_MAC(10_6);
 - (void)scrollIntoView:(BOOL)alignWithTop WEBKIT_AVAILABLE_MAC(10_5);
 - (void)scrollIntoViewIfNeeded:(BOOL)centerIfNeeded WEBKIT_AVAILABLE_MAC(10_5);
-- (void)scrollByLines:(int)lines WEBKIT_AVAILABLE_MAC(10_5);
-- (void)scrollByPages:(int)pages WEBKIT_AVAILABLE_MAC(10_5);
```

Comment:
This looks like it's public API given the WEBKIT_AVAILABLE_MAC declaration so we most likely cannot remove this. Given that nobody has touched this file I think that's very likely to be true.

I think we can probably annotate these methods with WEBKIT_CLASS_DEPRECATED_MAC instead and noop them.

Reply:
I think I will close the PR. :)